SLHCUpgradeSimulations/Configuration/python/HCalCustoms.py

```diff
@@ -247,3 +250,10 @@ def customise_Validation(process):
 
 def customise_condOverRides(process):
     return process
+    
+def customise_mixing(process):
```

Comment:
These customization functions are only needed for near-term development. Once we have the new packer and emaps, all of these customizations will be obviated. Since these things are expected on a relatively short timescale (hopefully ~weeks), I was thinking it was pointless to pollute the Eras and various python config files with such temporary customizations.